packages/react-heartwood-components/src/components/SprucebotTypedMessage/react-typing-animation/Typing.js

```diff
@@ -28,13 +34,64 @@ class Typing extends Component {
 	}
 
 	updateState = async state => {
-		if (this.hasMounted) {
-			return new Promise(resolve => {
-				this.setState(state, resolve)
-			})
+		if (!this.hasMounted) {
+			return
+		}
+		return new Promise(resolve => {
+			this.setState(state, resolve)
+		})
+	}
+
+	play = async () => {
```

Comment:
If you're getting into actively modifying this lib, you should consider porting it all to typescript; you've "broken the seal" so to say ;D

Reply:
I started to but it's a little too much for me right now, so I rolled back. 😞  Lots of monkey patching and unrolling of `any` args.